api/src/main/java/org/openmrs/Order.java

```diff
@@ -32,10 +32,18 @@
  * 
  * @version 1.0
  */
-public class Order extends BaseOpenmrsData implements java.io.Serializable {
+public class Order extends BaseOpenmrsData implements java.io.Serializable, Comparable{
 	
 	public static final long serialVersionUID = 4334343L;
-	
+
+	@Override
+	public int compareTo(Object o) {
```

Comment:
I understood what you mean.

But my understanding was only orders which have sequenceNumber set should be sorted within themselves to keep them in a consistent order,
other normal orders should remain as they are ordered.

Wont this change that ordering